src/app/components/chat/user-list/item/item.ts

```diff
@@ -142,6 +142,6 @@ export default class AppChatUserListItem extends Vue {
 			return;
 		}
 
-		leaveGroupRoom(this.chat);
+		leaveGroupRoom(this.chat, this.item);
```

Comment:
TS should've thrown an error here since item is either a chat room or a chat user.